<?xml version="1.0"?>
<launch>

	<param name="use_sim_time" value="true"/>
	<node name="NPG" pkg="grasp_learning" type="NPG.py" output="screen">
	 	<param name="input_output_data" value="$(find grasp_learning)/stored_data/training_data/task_dynamics/two_tasks_linear_dynamics_lambda_0.51.txt	" />
		<param name="load_model" value="0" />
		<param name="model_name" value="model.ckpt" />
 		<param name="num_inputs" value="2" />
		<param name="num_outputs" value="2" />
		<param name="num_rewards" value="1" />
		<param name="num_hidden_layers" value="1" />
		<rosparam param="hidden_layers_sizes">[2]</rosparam>
		<param name="use_static_learning_rate" value="1"/>
		<param name="learning_rate" value="0.5"/>
		<param name="subtract_baseline" value="0"/>
		<param name="batch_size" value="20" />
		<param name="discount_factor" value="0.99" />
		<param name="use_normalized_advantages" value="0" />
		<param name="relative_path" value="$(find grasp_learning)/stored_data/tested_data/" />
 	</node>
</launch>
